english/find-exit/chococopter/reeses/rpbc.md

```diff
@@ -0,0 +1,2 @@
+You find yourself transported in time back to a 1980 TV commercial for Reese's Peanut Butter Cups.
+Take yourself back to a much simpler time when there were people on the street walking, bell-bottoms and Sony Walk-Mans [enjoy this clip](https://www.youtube.com/watch?v=DJLDF6qZUX0)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.